test/pdfs/.gitignore

```diff
@@ -311,7 +311,6 @@
 !issue5567.pdf
 !issue5701.pdf
 !issue5896.pdf
-!issue5909.pdf
```

Comment:
Why is this one changed to a linked test case?

Reply:
*I should obviously have included this in the commit message, sorry about that!*

Because the *reduced* test-case is just two completely blank pages, and it's thus impossible to know if any future code changes breaks rendering (even though the file manages to load).

Reply:
Also, note that the PR which added that test was responsible for [the line that I'm trying to partially revert](https://github.com/mozilla/pdf.js/blob/c701ae3f4cd5f1713e09a1b0da0096664ca37f8f/src/core/obj.js#L1418) over in #11231 and I thus wanted to ensure proper test-coverage before making changes.

Reply:
Clear, thanks!